ambry-api/src/test/java/com.github.ambry/account/InMemAccountService.java

```diff
@@ -176,13 +177,11 @@ private Container getRandomContainer(short accountId) {
     boolean refContainerPreviousEncryption = refContainerEncryption || TestUtils.RANDOM.nextBoolean();
     boolean refContainerCaching = TestUtils.RANDOM.nextBoolean();
     boolean refContainerMediaScanDisabled = TestUtils.RANDOM.nextBoolean();
-    boolean refContainerTtlRequired = TestUtils.RANDOM.nextBoolean();
     return new ContainerBuilder(refContainerId, refContainerName, refContainerStatus, refContainerDescription,
         accountId).setEncrypted(refContainerEncryption)
         .setPreviouslyEncrypted(refContainerPreviousEncryption)
         .setCacheable(refContainerCaching)
         .setMediaScanDisabled(refContainerMediaScanDisabled)
-        .setTtlRequired(refContainerTtlRequired)
```

Comment:
explicitly set this to `false` (if someone changes the default in `Container` in the future)